test/ao/sparsity/test_activation_sparsifier.py

```diff
@@ -94,3 +212,25 @@ def test_activation_sparsifier(self):
 
         # Test Constructor
         self.check_constructor(activation_sparsifier, model, defaults, sparse_config)
+
+        # STEP 2: Register some layers
+        layer_args_list = _ActivationSparsifierUtils._get_register_layer_args(model)
+
+        # Registering..
+        for layer_args in layer_args_list:
+            layer_arg, sparse_config_layer = layer_args
+            activation_sparsifier.register_layer(**layer_arg, **sparse_config_layer)
+
+        # check if things are registered correctly
+        self.check_register_layer(activation_sparsifier, defaults, sparse_config, layer_args_list)
```

Comment:
also I'm wondering if we can separate these steps into multiple tests to keep the test itself shorter

Reply:
Since we're testing the complete workflow here, the state of sparsifier will change at each step. So, we have one single test-case here.